docs/rst/knora-ontologies/knora-base.rst

```diff
@@ -0,0 +1,1544 @@
+collection, along with a reification containing metadata about the link.
+We can visualise the result as the following graph:
+
+.. graphviz::
+
+   digraph {
+       {
+           // Resources
+           node [style = filled, fontcolor = white, color = navy, fillcolor = slateblue4]
+
+           work [label = "dali_4587"]
+           collection [label = "pompidou"]
+       }
+
+       {
+           // Values
+           node [style = filled, fontcolor = white, color = tomato3, fillcolor = tomato2]
+
+           title [label = "value_A"]
+           artistName [label = "value_B"]
+           collectionName [label = "value_C"]
+           linkValue [label = "value_D"]
+       }
+
+       {
+           // Literals
+           node [shape = box, color = firebrick]
+
+           titleStr [label = "‘The Persistence of Memory’"]
+           artistNameStr [label = "‘Salvador Dali’"]
+           isInCollection [label = "isInCollection"]
+           refCount [label = "1"]
+           collectionNameStr [label = "‘Centre Pompidou’"]
+       }
+
+       edge [fontsize = 11, color = cyan4]
+
+       work -> title [label = "hasTitle"]
+       work -> artistName [label = "hasArtist"]
+       work -> collection [label = "isInCollection"]
+       work -> linkValue [label = "isInCollectionValue"]
+
+       collection -> collectionName [label = "hasCollectionName"]
+
+       title -> titleStr [label = "valueHasString"]
+       artistName -> artistNameStr [label = "valueHasString"]
+       collectionName -> collectionNameStr [label = "valueHasString"]
+       linkValue -> refCount [label = "valueHasRefCount"]
+       linkValue -> work [label = "subject"]
+       linkValue -> isInCollection [label = "predicate"]
+       linkValue -> collection [label = "object"]
+    }
+
+.. _knora-base-standoff:
+
+Text with Standoff Markup
+-------------------------
+
+Knora is designed to be able to store text with markup, which can
+indicate formatting and structure, as well as the complex observations
+involved in transcribing handwritten manuscripts. One popular way of
+representing text in the humanities is to encode it in XML
+using the Text Encoding Initiative (`TEI`_) guidelines. [#]_ In Knora, a
+TEI/XML document can be stored as a file with attached metadata, but
+this is not recommended, because it does not allow Knora to perform
+searches across multiple documents.
+
+The recommended way to store text with markup in Knora is to use
+Knora's built-in support for “standoff” markup, which is stored
+separately from the text. This has some advantages over embedded markup
+such as XML. [#]_ While XML requires markup to have a hierarchical
+structure, and does not allow overlapping tags, standoff nodes do not
+have these limitations (Schmidt2016_). A standoff tag can be attached to
+any substring in the text by giving its start and end positions. [#]_
+For example, suppose we have the following text:
+
+    This sentence has overlapping visual attributes.
```

Comment:
This sentence should be formatted (see
![screen shot 2016-10-24 at 09 19 11](https://cloud.githubusercontent.com/assets/6000023/19636662/ffce7574-99ca-11e6-92ef-e751695db5f3.png)
)